### ISSUE TITLE
Auto creating pages upon activation causes WP_DEBUG message after de- and re-activated

### ISSUE THREAD

**Issue by cliffordp** [pippinsplugins/Restrict-Content-Pro#157]
These WP_DEBUG notices appeared after deactivating and reactivating RCP:
- `Notice: Undefined index: free_message in /wp-content/plugins/restrict-content-pro/includes/content-filters.php on line 235`
- `Notice: Undefined index: paid_message in /wp-content/plugins/restrict-content-pro/includes/content-filters.php on line 236`

I only had 2 pages (auto-created from initial RCP activation: Register and Welcome).
After reactivating, I noticed I had 4 pages (duplicates, now post IDs of course)
I deleted these 2 pages, went back to RCP Settings and saved and these 2 WP_DEBUG messages went away.

So I'd like to see this fixed, whether it's NOT auto-creating 2 new pages upon every activation and/or no WP_DEBUG messages.

Reference: https://github.com/pippinsplugins/Restrict-Content-Pro/issues/116

**Comment by misulicus** [comment]
got the same issue with new pages created on each activation and also these in debug.log:
```
[21-May-2015 15:48:34 UTC] WordPress database error Duplicate column name 'max_uses' for query ALTER TABLE `dev_rcp_discounts` ADD `max_uses` mediumint made by activate_plugin, do_action('activate_Restrict-Content-Pro-master/restrict-content-pro.php'), call_user_func_array, rcp_options_upgrade

[21-May-2015 15:48:34 UTC] WordPress database error Duplicate column name 'expiration' for query ALTER TABLE `dev_rcp_discounts` ADD `expiration` mediumtext made by activate_plugin, do_action('activate_Restrict-Content-Pro-master/restrict-content-pro.php'), call_user_func_array, rcp_options_upgrade

[21-May-2015 15:48:34 UTC] WordPress database error Duplicate column name 'subscription_id' for query ALTER TABLE `dev_rcp_discounts` ADD `subscription_id` mediumint made by activate_plugin, do_action('activate_Restrict-Content-Pro-master/restrict-content-pro.php'), call_user_func_array, rcp_options_upgrade

[21-May-2015 15:48:34 UTC] WordPress database error Duplicate column name 'transaction_id' for query ALTER TABLE `dev_rcp_payments` ADD `transaction_id` tinytext made by activate_plugin, do_action('activate_Restrict-Content-Pro-master/restrict-content-pro.php'), call_user_func_array, rcp_options_upgrade

[21-May-2015 15:48:34 UTC] WordPress database error Duplicate column name 'status' for query ALTER TABLE `dev_rcp_payments` ADD `status` varchar(200) made by activate_plugin, do_action('activate_Restrict-Content-Pro-master/restrict-content-pro.php'), call_user_func_array, rcp_options_upgrade
```

**Comment by pippinsplugins** [comment]
Thanks, I'll get these resolved for 2.1.8

**Comment by pippinsplugins** [comment]
Partially resolved in #187

**Comment by pippinsplugins** [comment]
Duplicate pages issue has been resolved as well.
